docs/configuration/vpn/wireguard.rst

```diff
@@ -222,6 +287,24 @@ Operational Commands
 Status
 ======
 
+.. opcmd:: sudo wg
```

Comment:
@c-po what do you think about this? It is not really an operation command. I have doubts to official mention sudo commands.

Reply:
Regardless of its position in the doc, I think it should still be present. It provides diagnostics such as the latest handshake and traffic rx & tx, that isn't available through any of the other commands.

Reply:
We do not document such raw commands. If the diagnostic data is usefule, we should rather add a new `show interfaces wireguard wg0 detail` command

Reply:
Sure, will commit later.